tcms_tenants/context_processors.py

```diff
@@ -0,0 +1,16 @@
+from django.template.loader import render_to_string  # noqa: E402
+
+
+def tenant_navbar_processor(request):
+    """
+    Provide tenant name for display in navbar
+    """
+    tenant_name = request.tenant.schema_name
+    if request.tenant.organization:
+        tenant_name = request.tenant.organization
+    if request.tenant.name:
+        tenant_name = request.tenant.name
```

Comment:
This will unconditionally override the value coming from `tenant.organization`. We want to use the name only when organization isn't configured.